### PR TITLE
David

### PR DESCRIPTION
**Pull request by DavidBarke**

### CODE REVIEW

File: man/mixmod_regression.Rd [hunk 1]
```diff
@@ -57,22 +57,21 @@ subgroups. See 'Details'.
...
+\code{\link[segmented:segmented]{segmented.lm}}.
 
 Since the attempt to separate the failure data happens in an automated fashion,
 the algorithm tends to overestimate the number of breakpoints (see 'Warning'
-in \code{\link[segmented:segmented]{segmented}}).
+in \code{\link[segmented:segmented]{segmented.lm}}).
 
 In the context of reliability analysis it is important that the main types of
-faults can be identified and analyzed separately. These are
+failure can be identified and analyzed separately. These are
```
Comment (Tim-TU) [inline man/mixmod_regression.Rd:67]: As in .R mentioned I would use plural

File: man/plot_layout.Rd [hunk 2]
```diff
@@ -35,7 +35,8 @@ Returns a plotly object which contains the layout
   that is used for probability plotting.
 }
 \description{
-This function is used to create the layout of a probability plot.
+This function is used to create the layout of a probability plot. It is called
+by \code{\link{plot_prob}} internally.
```
Comment (Tim-TU) [inline man/plot_layout.Rd:39]: The function is called inside plot_prob to determine appearance of the grid with respect to the given x
... I think some more information would be nice at this point

File: man/plot_layout.Rd [hunk 3]
```diff
@@ -35,7 +35,8 @@ Returns a plotly object which contains the layout
   that is used for probability plotting.
```
Comment (Tim-TU) [inline man/plot_layout.Rd:35]: Could you please check the doc of plot_layout? I think "returns a plotly object" is written there...

File: man/plot_mod.Rd [hunk 4]
```diff
@@ -3,13 +3,28 @@
...
+\alias{plot_mod.mixmod_em}
 \title{Add Estimated Population Line(s) to a Probability Plot}
 \usage{
-plot_mod(p_obj, x, ...)
+## S3 method for classes 'model_estimation', 'model_estimation_list',
+## 'mixmod_regression', 'mixmod_regression_list', 'mixmod_em'
+plot_mod(p_obj, x, title_trace = "Fit")
+}
+\arguments{
+\item{p_obj}{A plot object returned from \code{\link{plot_prob}}.}
```
Comment (Tim-TU) [inline man/plot_mod.Rd:18]: Maybe returned by

**Review by Tim-TU — APPROVED** [review]

File: man/plot_mod.Rd (hunk 4 above)
Comment (DavidBarke) [reply man/plot_mod.Rd:18]: At the moment "returned from" is the consistent expression. We'd have to change this at many locations, if necessary?

**Pull request merged**